### PR TITLE
virttest/libvirt_vm: check the status of qemu-guest-agent

### PR DESCRIPTION
**Pull request by PandaWei**
add steps to check the status of qemu-guest-agent service
after starting or stopping it.

### CODE REVIEW

File: virttest/libvirt_vm.py [hunk 1]
```diff
@@ -2315,6 +2315,9 @@ def prepare_guest_agent(self, prepare_xml=True, channel=True, start=True):
         self.install_package('qemu-guest-agent')
 
         session = self.wait_for_login()
+
+        def _is_qa_running():
```
Comment (Hao-Liu) [inline virttest/libvirt_vm.py]: We usually abbreviate qemu-guest-agent as qemu-ga or simply ga. I don't think qa is a intuitive name.

File: virttest/libvirt_vm.py [hunk 2]
```diff
@@ -2325,6 +2328,10 @@ def prepare_guest_agent(self, prepare_xml=True, channel=True, start=True):
             if status != 0:
                 raise virt_vm.VMError("Start/stop of qemu-guest-agent "
                                       "failed:\n%s" % output)
+            # Check qemu-guest-agent status
+            if utils_misc.wait_for(_is_qa_running, timeout=60):
```
Comment (Hao-Liu) [inline virttest/libvirt_vm.py]: What if stop command is issue but qemu-ga process not exit immediately? Maybe we should also wait for qemu-ga exit for a specific time as what you do to wait for start.

Comment (PandaWei) [inline virttest/libvirt_vm.py]: Yes, your suggestion sounds good.
I will update as it.

**Comment by Hao-Liu** [comment]
Now this PR is good to me. Thanks @PandaWei.

**Comment by waynesun09** [comment]
ACK

**Pull request merged**